plugins/drng/parameters.go

```diff
@@ -0,0 +1,19 @@
+package drng
+
+import (
+	flag "github.com/spf13/pflag"
+)
+
+const (
```

Comment:
Lets chance configuration keys to be Go conform by naming them like `CfgDRNGInstanceID`/`CfgDRNGThreshold` etc.